cdn-links.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: CDN Links
+---
+
+## Polybase Client
+
+A client to connect to the Polybase decentralized database.
+
+Install: 
+
+```html
+<script src = 'https://cdn.jsdelivr.net/npm/@polybase/client@0.3.22/dist/bundle.min.js'></script>
+```
+
+Usage:
+
+```js
+// Access methods with global variable 'polybase'
+const db = polybase.client({
+  defaultNamespace: "your-namespace",
+})
+```
+
+## Polybase Eth
+
+A set of ethereum helpers based on EIP-1193, to make signing requests easier on Polybase.
+
+Install: 
+
+```html
+<script src = 'https://cdn.jsdelivr.net/npm/@polybase/eth@0.3.22/dist/bundle.min.js'></script>
```

Comment:
Can we actually remove the patch version, so it would be `https://cdn.jsdelivr.net/npm/@polybase/eth@0.3/dist/bundle.min.js`

Same would need to be applied to the others